Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -1303,6 +1303,20 @@ public static partial class PythonOps {
             return v;
         }
 
+        public static bool TryFixIndex(int v, int len, out int res) {
```

Comment:
Should we deduplicate the code in FixIndex with this?

Reply:
Might as well, would make it nicer. I would also make `TryFixIndex` `internal` unless we have a good reason not to. The less public API surface there is the easier the codebase is to maintain.